lib/webdriver.py

```diff
@@ -247,8 +249,7 @@ def _handle_login_error(self, response: JSON) -> LoginError:
     def _get_needed_headers(self, request_headers: JSON) -> JSON:
         headers = {}
         for header in request_headers:
-            if re.match(r"x-api-key|x-channel-id|user-agent|^[\w-]+?-\w$", header, re.I):
-                headers[header] = request_headers[header]
+            headers[header] = request_headers[header]
```

Comment:
Do more headers need to be matched? If all of the headers are saved, this function isn't needed. However, it's unclear to me whether or not all request headers are actually relevant to the script bypassing 403s.